core/playwright/externalFormInvite.spec.ts

```diff
@@ -109,7 +109,8 @@ test.describe("Inviting a new user to fill out a form", () => {
 		await runActionDialog.getByRole("button", { name: "Close", exact: true }).click();
 		await runActionDialog.waitFor({ state: "hidden" });
 	});
-	test("New user can fill out the form from the email link", async ({ browser }) => {
+	// fails with large number of pubs in the db
+	test.fixme("New user can fill out the form from the email link", async ({ browser }) => {
```

Comment:
yeah I would kind of like the keep the autosave test running for now, since I think we could conceivably break that in the near future. so maybe we can just figure out how to make sure the big seed doesn't happen in CI? or at least can we just disable the autosave expectation instead of the whole test?

Reply:
it failed bc the `MINIMAL_SEED` was only set in `globalSetup.ts`, which is not run by playwright (only by vitest). e2e tests were seeded manually in `e2e.yml`. I set `MINIMAL_SEED` to true there and reenabled this test!